recipe/meta.yaml

```diff
@@ -1,32 +1,35 @@
-{% set version = "0.7.3" %}
+{% set version = "0.8.0" %}
 
 package:
   name: hvplot
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/h/hvplot/hvplot-{{ version }}.tar.gz
-  sha256: 74b269c6e118dd6f7d2a4039e91f16a193638f4119b4358dc6dbd58a2e71e432
+  sha256: 8630dba34969b105e267cbe14237e2d56d9969c801980ae5e5190bc18b2792d6
 
 build:
-  number: 1
-  noarch: python
+  number: 0
+  # ModuleNotFoundError: No module named 'hvplot.__main__'
+  # entry_points:
+  #   - hvplot = hvplot.__main__:main
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - python
+    - pyct
     - pip
-    - wheel
     - setuptools
-    - pyct >=0.4.4
-    - param >=1.6.1
+    - wheel
   run:
     - python
-    - bokeh >=1.0.0
+    - packaging
+    - param >=1.12.0
```

Comment:
`param >=1.6.1`
https://github.com/holoviz/hvplot/blob/v0.8.0/setup.py#L101

Reply:
`param >=1.6.1` should be not in `run` but in `host`

Reply:
@varlackc The maintainers have pinned to `param >=1.12.0`  in the [conda-forge](https://github.com/conda-forge/hvplot-feedstock/commit/92ab82cd9dc23e697a1f7b1fe8757954062ed87e#) recipe which they also maintain. See Philip's comment above - they are in favor of using the stricter pinnings.

@skupr-anaconda Could you please explain why it should be in host?

Reply:
When building panel from source should be both in `host` and in `run` but since this is using an existing wheel I guess the `host` isn't needed. Suspect `pyct` isn't needed in host for the same reason.

Reply:
param is inside extra_requires[build]. It's equal to host for python packages. If the maintainers don't mean something else

Reply:
> but since this is using an existing wheel 

@philippjfr - here we are actually building from the [source file](https://github.com/AnacondaRecipes/hvplot-feedstock/blob/20c3b4b39a5aa3d46c90211a2da77a1cf52f242b/recipe/meta.yaml#L8). 

I looked up the `panel-feedstock` and saw [this section](https://github.com/conda-forge/panel-feedstock/blob/83252fe5f3ac32fef48c5389f081cf5fea45779f/recipe/meta.yaml#L24-L29) with the comment `# These are also needed at build time.` - and `param` is listed under that.

So is `param` also required to _build_ `hvplot`? If so, we can add it to `host`. If not, I believe just having it under `run` will be fine.

Reply:
Ah, okay, yes it that case it's needed.

Reply:
Great made those amendments!